src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/api/PublicApiResource.kt

```diff
@@ -21,7 +21,7 @@ class PublicApiResource(private val locationService: LocationMappingService) {
 
   @PreAuthorize("hasRole('ROLE_NOMIS_DPS_MAPPING__LOCATIONS__R')")
   @GetMapping("/locations/nomis/{nomisLocationId}")
-  @Tag(name = "NOMIS / DPS Mapping lookup")
+  @Tag(name = "NOMIS to DPS Mapping lookup")
```

Comment:
I changed this recently because it includes DPS to Nomis as well as v.v. and the tag should be the same for both methods (otherwise they get put in separate sections